elastic/kibana
# [ObsUx] [Infra] Change container details view with asset details view

Part of https://github.com/elastic/kibana/issues/179844

### In this PR
- From Inventory, open asset details page view for Containers
- Show overview tab with CPU and Memory KPIs and metric charts
- Metadata tab with old fields, more metadata fields will be shown in follow-up PR
- Added links to container metrics documentation, currently there are no docs for K8s metrics just for docker containers

#### How to test
- The feature is under a FF, on inventory page go to settings and enable `Container view`
- In containers inventory, select a container and click on 'Docker container metrics' link (there's an [issue](https://github.com/elastic/kibana/issues/180806) to reword this links as K8s containers are also shown)
- Container details page should be shown with overview and metadata tabs
- On overview tab KPIs for CPU and Memory and Metrics section with CPU and Memory charts should be displayed
<img width="937" alt="image" src="https://github.com/elastic/kibana/assets/31922082/d2f25f2a-ea4f-4516-b216-38464682fd14">
